### PR TITLE
PR Pack

### PR DESCRIPTION
**Pull request by danimesq**
This PR is a joint of all of my recent branches with the following features:

- [x] - "--auto" flag to automatically guess OS' path and the --latest chat (+ the --recent workspace)!
- [x] - Show the AI model used in a message
X Install system-wide
- [x] Don't create empty `images/` folder when there were no images

### CODE REVIEW

**Comment by danimesq** [comment]
@somogyijanos 

> Hm, getting confused a bit. I really like the option to export all or the latest tab from the most recent workspace. I also like automatically detecting the path to the vscdb file. But I think we don't need both separately, I mean detecting the path automatically makes only sense if we don't want to input any path, this also means we're expecting to export from the most recent workspace (either the latest tab or all of them). So my suggestion to make it less confusing:
> 
> discover:
> 
>     make the db_path argument optional, if not provided, it will be determined automatically
> 
> export:
> 
>     make the db_path argument optional, if not provided, it will be determined automatically to be the latest workspace
>     --latest-tab/ --all-tabs boolean flag pair for specifying that only the latest tab gets exported or all tabs (default)
>     usage
>         export all tabs from a specific workspace: export /path/to/db.vscdb
>         export latest tab from a specific workspace: export --latest-tab /path/to/db.vscdb
>         export all tabs from latest workspace: export
>         export latest tab from latest workspace: export --latest-tab

Did you also mean to remove the `--auto` flag?

**Comment by somogyijanos** [comment]
Yep, I'd cover this functionality by making the path argument optional for both commands, so if it's not provided it means it's determined automatically. For `discover` this means the base path, for `export` the latest workspace folder.

**Comment by danimesq** [comment]
@somogyijanos are the latest commit covering all?

File: install.sh
Comment (somogyijanos) [inline install.sh:1]: @danimesq I'm not sure yet I want an installation like this, please put it at least into another PR until I think about how the installation would be best. Thanks!

Comment (somogyijanos) [reply install.sh:1]: Please adjust the README.md accordingly.

**Review by somogyijanos — CHANGES_REQUESTED** [review]
👆🏻

Everything else looks super good to me!

**Comment by danimesq** [comment]
@somogyijanos done in: https://github.com/somogyijanos/cursor-chat-export/pull/3/commits/1969b9990e45e25371dd7fde9ec5f2e54788660d

File: README.md [hunk 1]
```diff
@@ -13,44 +13,20 @@ Also see [this](https://forum.cursor.com/t/guide-5-steps-exporting-chats-prompts
 
 ## Installation
 
-### Option 1: Manual Installation
-
 1. Clone the repository:
     ```sh
-    git clone https://github.com/somogyijanos/cursor-chat-export.git
-    cd cursor-chat-export
+    git clone https://github.com/somogyijanos/python3 chat.py.git
```
Comment (somogyijanos) [inline README.md:18]: I think here you replaced `cursor-chat-export` by accident ;)

File: README.md [hunk 2]
```diff
@@ -26,7 +26,9 @@ Also see [this](https://forum.cursor.com/t/guide-5-steps-exporting-chats-prompts
 
 ## Usage
 
-Find, where the `state.vscdb` is located in your computer. The table below may help:
+If you used the automated installation script, you can use the `python3 chat.py` command instead of `./chat.py` in the following examples. This is useful for quickly exporting the chat on your current project folder, by simply opening the terminal on it instead of having to copy python3 chat.pyer every time.
```
Comment (somogyijanos) [inline README.md:29]: This part isn't updated.

File: README.md [hunk 3]
```diff
@@ -36,10 +38,43 @@ Find, where the `state.vscdb` is located in your computer. The table below may h
...
 ```
 
+If no directory is provided, the tool will automatically use the default Cursor workspace storage directory for your OS.
+
 ### Export Chats of a Workspace
 ```sh
-./chat.py export --output-dir output "/Users/myuser/Library/Application Support/Cursor/User/workspaceStorage/b989572f2e2186b48b808da2da437416/state.vscdb"
-```
\ No newline at end of file
+python3 chat.py export --output-dir output "/Users/myuser/Library/Application Support/Cursor/User/workspaceStorage/b989572f2e2186b48b808da2da437416/state.vscdb"
```
Comment (somogyijanos) [inline README.md:48]: Here's a lot of things doubled, please update.

**Review by somogyijanos — CHANGES_REQUESTED** [review]
Some changes needed in `README.md`.

**Comment by danimesq** [comment]
@somogyijanos hopefully fixed now.

**Comment by somogyijanos** [comment]
I spent some time with these new features and did some refact, see #5. Thanks @danimesq for your contrib!

**Pull request merged**